### PR TITLE
Non-Parametric One Way Variance

### PR DESCRIPTION
**Pull request by nyagishira**
Still working on it

### CODE REVIEW

**Comment by volloholic** [comment]
To discuss

**Comment by nyagishira** [comment]
i have submitted code

**Comment by nyagishira** [comment]
i have submitted code

File: instat/dlgNon-ParametricANOVA.vb [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+
+    End Sub
+    Private Sub FillFormular()
+        Dim strYVariate As String
+        Dim strFactor As String
+
+        strYVariate = ucrReceiverYVariate.GetVariables()
+        strFactor = ucrReceiverFactor.GetVariables()
+        If ((Not IsNothing(strYVariate)) Or ((Not IsNothing(strFactor)))) Then
+            ucrBase.clsRsyntax.AddParameter("formula", strYVariate & "~" & strFactor)
```
Comment (volloholic) [inline instat/dlgNon-ParametricANOVA.vb]: is this or or and?

**Pull request closed without merging**